data/json/furniture.json

```diff
@@ -2769,5 +2769,32 @@
                 { "item": "rock", "count": [18, 30] }
             ]
         }
+    },{
+        "type" : "furniture",
+        "id" : "f_brazier",
+        "name": "brazier",
+        "symbol": "#",
+        "color": "red",
+        "move_cost_mod": 2,
+        "required_str": 16,
+        "flags": ["PLACE_ITEM", "TRANSPARENT", "FIRE_CONTAINER"],
+        "deconstruct": {
+            "items": [
+                { "item": "sheet_metal", "count": [0, 2] },
+                { "item": "scrap", "count": [2, 3] },
+                { "item": "steel_chunk", "count": [1, 2] }
+            ]
```

Comment:
I wasn't sure what path is better - deconstruct it to brazier item or directly to components. Now that you mention it, I'll change it.